src/ia/js/IAPage.js

```diff
@@ -55,28 +55,27 @@
                         latest_edits_data = ia_data.live;
                     }
 
-                    if (ia_data.live.dev_milestone !== "live") {
-                        if ($(".special-permissions").length) {
-                            ia_data.permissions = {can_edit: 1};
+
+                    if ($(".special-permissions").length) {
+                        ia_data.permissions = {can_edit: 1};
                             
-                            if ($("#view_commits").length) {
-                                ia_data.permissions.admin = 1;
-                            }
+                        if ($("#view_commits").length) {
+                            ia_data.permissions.admin = 1;
                         }
-                        ia_data.current = {};
-                        ia_data.current[ia_data.live.dev_milestone] = 1;
-                        var future = {};
-                        var milestone_idx = $.inArray(ia_data.live.dev_milestone, page.dev_milestones_order);
-                        if (milestone_idx !== -1) {
-                            milestone_idx++;
-                            for (var i = milestone_idx; i < page.dev_milestones_order.length; i++) {
-                                future[page.dev_milestones_order[i]] = 1;
-                            }
+                    }
```

Comment:
Sorry for explaning myself poorly, I meant just remove the check for this permissions part. The code afterwards, which sets the current and future dev milestone, isn't needed for live pages, so could you put the check back for lines https://github.com/duckduckgo/community-platform/blob/jag/screenshot-button/src/ia/js/IAPage.js#L66-L77 ?